index.md

```diff
@@ -1,9 +1,34 @@
-## Infrastructure services
+Intended Audience: infrastructure team developers
```

Comment:
It would be fine to have a heading for this document.  I think it would even be fine for the file to be named something other than "index"